diff -Naur buildroot.orig/package/mpd/mpd.conf buildroot/package/mpd/mpd.conf
--- buildroot.orig/package/mpd/mpd.conf	2017-03-03 12:35:35.746932480 +0100
+++ buildroot/package/mpd/mpd.conf	2017-03-03 12:57:55.862897352 +0100
@@ -1,32 +1,30 @@
-#
 # Sample configuration file for mpd
 # This is a minimal configuration, see the manpage for more options
 #
 
 # Directory where the music is stored
-music_directory		"/var/lib/mpd/music"
+music_directory				"/media/nas/music"
 
 # Directory where user-made playlists are stored (RW)
-playlist_directory	"/var/lib/mpd/playlists"
+playlist_directory			"/usr/local/mpd/playlists"
 
 # Database file (RW)
-db_file			"/var/lib/mpd/database"
+db_file						"/usr/local/mpd/database"
 
 # Log file (RW)
-log_file		"/var/log/mpd.log"
+log_file					"/var/log/mpd.log"
 
 # Process ID file (RW)
-pid_file		"/var/run/mpd.pid"
+pid_file					"/var/run/mpd.pid"
 
 # State file (RW)
-state_file		"/var/lib/mpd/state"
+state_file					"/usr/local/mpd/state"
 
-# User id to run the daemon as
-#user			"nobody"
 
-# TCP socket binding
-bind_to_address		"any"
-#bind_to_address	"localhost"
+bind_to_address				"any"
 
-# Unix socket to listen on
-bind_to_address		"/var/lib/mpd/socket"
+audio_output {
+        type            "alsa"
+        name            "ALSA device @raspi"
+        device          "hw:0,0"        # optional
+}
diff -Naur buildroot.orig/package/mpd/S95mpd buildroot/package/mpd/S95mpd
--- buildroot.orig/package/mpd/S95mpd	2017-03-03 12:35:35.886932476 +0100
+++ buildroot/package/mpd/S95mpd	2017-03-03 13:02:45.314889764 +0100
@@ -4,16 +4,23 @@
 test -f /usr/bin/mpd || exit 0
 test -f /etc/mpd.conf || exit 0
 
+PIDFILE="/var/run/mpd.pid"
+MPDDIR="/usr/local/mpd"
+
 start() {
+	echo -n "Loading sound driver for mpd: "
+	modprobe snd-bcm2835
+	echo "OK"
 	printf "Starting mpd: "
-	start-stop-daemon --start --quiet --background --exec /usr/bin/mpd \
-		&& echo "OK" || echo "FAIL"
+	if [ ! -e "$MPDDIR" ] ; then
+		mkdir -p $MPDDIR/playlists
+	fi
+	start-stop-daemon --start --quiet --background --exec /usr/bin/mpd && echo "OK" || echo "FAIL"
 }
 
 stop() {
 	printf "Stopping mpd: "
-	start-stop-daemon --stop --quiet --pidfile /var/run/mpd.pid \
-		&& echo "OK" || echo "FAIL"
+	start-stop-daemon --stop --quiet --pidfile $PIDFILE && echo "OK" || echo "FAIL"
 }
 
 case "$1" in
